examples/bpk-component-datatable/examples.js

```diff
@@ -16,6 +16,8 @@
  * limitations under the License.
  */
 
+/* eslint-disable react/prop-types */
```

Comment:
I guess this is temporary due to we are not yet migrating our examples to TS and still using flow things here?

Reply:
Yes, we can remove this when we migrate to TS